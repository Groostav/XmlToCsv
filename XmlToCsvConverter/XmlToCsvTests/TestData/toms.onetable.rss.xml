<?xml version="1.0" encoding="UTF-8"?>
<rss version="2.0" xmlns:dc="http://purl.org/dc/elements/1.1/">
		<title><![CDATA[Reviews Tom's Hardware]]></title>
		<description><![CDATA[Tom's Hardware]]></description>
		<link>http://www.tomshardware.com</link>		
		<language>en</language>
		<ttl>10</ttl>
		<lastBuildDate>Thu, 30 Jan 2014 08:39:46 +0100</lastBuildDate>
</rss>