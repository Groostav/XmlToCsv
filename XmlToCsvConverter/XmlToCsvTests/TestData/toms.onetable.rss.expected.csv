title,description,link,language,ttl,lastBuildDate,version
"Reviews Tom's Hardware","Tom's Hardware","http://www.tomshardware.com","en","10","Thu, 30 Jan 2014 08:39:46 +0100","2.0"
